### ISSUE TITLE
System Status Page

### ISSUE THREAD

**Issue by majgis** [nodesecurity/nsp#99]
A system status page like github has would be a welcome addition:
https://status.github.com/graphs/past_month

It would put numbers behind changes associated with #98 and would build confidence in relying on the service.

**Comment by pdehaan** [comment]
Or even something like npm's http://status.npmjs.org/ (which seems to use https://www.statuspage.io/ -- although I can't see if they have any open-source plans, or if the cheapest plan is their $30/mo tier).

**Comment by evilpacket** [comment]
yes, agreed. We currently use uptime robot. Not sure if we can expose that or not, but we should probably look into a status page. Here is our stats for the last 24 hours. Not good, but we have a plan to make things better.

![](https://cldup.com/jg1xnUaRxO.png)

**Comment by RavenHursT** [comment]
Getting intermittent errors on our builds today: 

> 'nsp-check' errored after 50 s
> Got an invalid response from Node Security, please email the above debug output to support@nodesecurity.io

Would be awesome if this request was made reality since we're not sure if there's something wrong w/ our network to NSP, or something wrong on NSP's side causing the 50s timeout...
